docs/source/en/model_doc/owlvit.mdx

```diff
@@ -39,19 +39,26 @@ OWL-ViT is a zero-shot text-conditioned object detection model. OWL-ViT uses [CL
 
 >>> url = "http://images.cocodataset.org/val2017/000000039769.jpg"
 >>> image = Image.open(requests.get(url, stream=True).raw)
-
->>> inputs = processor(text=[["a photo of a cat", "a photo of a dog"]], images=image, return_tensors="pt")
-
+>>> texts = [["a photo of a cat", "a photo of a dog"]]
```

Comment:
The prompts for zero-shot object detection are typically just the classes, right? LIke "cat", "remote", "couch", etc.

Reply:
Not necessarily, they use simple words as text queries in their demo but the backbone and the object detection model is trained using the same text template augmentations as CLIP

I use simple words in the colab demo to replicate the original demo but I think it's an important part of the paper

Reply:
Ok, because I just checked their Colab demo notebook, they use:

> text_queries = ['human face', 'rocket', 'nasa badge', 'star-spangled banner']

=> wouldn't performance be better when leaving out "a photo of a ..." as the fine-tuned model didn't see that?

Reply:
Like I said, they use simple words in their demo but both they augment the data with text templates during the training of the CLIP backbone and fine-tuning the detection model

You can find the relevant part in Appendix A1.9 of the paper (also Fig. A1), where they show prompting during training and testing yields better results

Reply:
Sorry, should have read that better!

Thanks for explaining!

Reply:
No problem, it's kind of squished into the paper :)